src/CompactErrorFormatter.php

```diff
@@ -48,7 +53,8 @@ public function formatErrors(
 					$this->format,
 					[
 						'{absolutePath}' => $absolutePath,
-						'{path}' => $this->relativePathHelper->getRelativePath($fileSpecificError->getFile()),
+                        '{editorUrl}' => str_replace(['%file%', '%line%'], [$absolutePath, $fileSpecificError->getLine() ?? ''], $this->editorUrl),
```

Comment:
Please extract into closure/function as parametr expansion can be separated:

```php
$formatEditorUrl = fn(string $path, string $line) => str_replace(['%file%', '%line%'], [$path, $line], $this->editorUrl);

'{editorUrl}' => $formatEditorUrl($absolutePath, (string) ($fileSpecificError->getLine() ?? ''));
```

Reply:
I decided to not use this package anymore, and use my own formatter because I wanted to do more modifications. Therefore I have no interest in updating the PR anymore. Should I close it? Or do we merge this as is?

Reply:
I also don't agree with that the proposed change is a better one tbh.

Reply:
What are those modifications? Feel free to open another PR for them.

Reply:
Color highlighting https://github.com/phpstan/phpstan-src/pull/841#issuecomment-991957024
Very opinionated and not in the scope of this library I would say.

Reply:
I have no problem with keeping this the way it is. I idea here is to keep the formatting logic separated. No big deal.